VotingApplication/VotingApplication.Web/Scripts/Services/CreateService.js

```diff
@@ -0,0 +1,44 @@
+﻿(function () {
+    'use strict';
+
+    angular
+        .module('VoteOn-Create')
+        .factory('CreateService', CreateService);
+
+    CreateService.$inject = ['$http', '$q', 'AccountService'];
+
+    function CreateService($http, $q, AccountService) {
+        
+        var service = {
+            createPoll: createPoll
+        };
+
+        return service;
+
+        function createPoll(poll) {
+            var token;
+            if (AccountService.account) {
+                token = AccountService.account.token;
+            }
+            else {
+                token = null;
+            }
+
+            var prom = $q.defer();
```

Comment:
This isn't needed here, as `then` returns a new promise. Lines 38-41 can be written as 
```javascript
.then(function(response){
return response.data;
});
```